token_benchmark_ray.py

```diff
@@ -74,11 +74,9 @@ def get_token_throughput_latencies(
     """
     random.seed(11111)
 
-    tokenizer = LlamaTokenizerFast.from_pretrained(
-        "hf-internal-testing/llama-tokenizer"
-    )
-    get_token_length = lambda text: len(tokenizer.encode(text))
-
+    # Use same tokenizer with furiosa-llm serve
+    tokenizer = PreTrainedTokenizerFast.from_pretrained("meta-llama/Meta-Llama-3.1-70B-Instruct")
```

Comment:
인자로 전달이 가능하면 좋을 것 같은데요, 마침 `--model` 인자가 뚫려있으니 (저희가 그동안 no-op 인자로 사용하고 있었지만) 아래와 같은 방향은 어떨까요? 
```suggestion
    tokenizer = PreTrainedTokenizerFast.from_pretrained(model)
```

Reply:
반영했습니다 [122c0aa](https://github.com/furiosa-ai/llmperf/pull/3/commits/122c0aa4e28ca5629c1f17be13d2cad93b46931b)

Reply:
앗 뒷북 죄송한데 위에 들어간 주석도 지워주시면 감사드리겠습니다. (`# Use same tokenizer with furiosa-llm serve`)

Reply:
넵! [a896f9b](https://github.com/furiosa-ai/llmperf/pull/3/commits/a896f9be33122914422e2056e1c980493a8f718b)